test/go/state_tracker_test.go

```diff
@@ -38,6 +38,19 @@ func (s *StateTrackerTestSuite) TestMinBinlogPosition() {
 		},
 	}
 	s.Require().Equal(serializedState.MinBinlogPosition(), mysql.Position{"mysql-bin.00002", 10})
+
+	serializedState = &ghostferry.SerializableState{
+		LastWrittenBinlogPosition: mysql.Position{
+			Name: "",
+			Pos:  0,
+		},
+
+		LastStoredBinlogPositionForInlineVerifier: mysql.Position{
+			Name: "mysql-bin.00002",
+			Pos:  10,
+		},
+	}
+	s.Require().Equal(serializedState.MinBinlogPosition(), mysql.Position{"mysql-bin.00002", 10})
```

Comment:
This is a good case to cover, but with regards to the changes in this PR, I would have expected the `LastStoredBinlogPositionForInlineVerifier` to be empty and for `LastWrittenBinlogPosition` to have the values of `mysql.Position{"mysql-bin.00002", 10}` given we're now only setting `LastStoredBinlogPositionForInlineVerifier` if the `InlineVerifier` is being used.